src/work-allocation/containers/available-tasks/available-tasks.component.html

```diff
@@ -3,4 +3,4 @@
   [taskServiceConfig]="taskServiceConfig" [tasks]="tasks" [fields]="fields"
   [sortedBy]="sortedBy" [emptyMessage]="emptyMessage"
   (sortEvent)="onSortHandler($event)" (actionEvent)="onActionHandler($event)">
-</exui-task-list>
+</exui-task-list>
```

Comment:
_Nitpick:_
Why remove the line feed at the end?

Reply:
Done

Reply:
Change not made to workAllocation1